assets/shop/js/mollie/app.js

```diff
@@ -6,34 +6,35 @@ document.addEventListener('DOMContentLoaded', function () {
     let mollieData = document.querySelector('.online-online-payment__container');
```

Comment:
How about returning early in here if there are no payments on the page?
It should simplify the following scripts.